data/hai/hai missions.txt

```diff
@@ -65,8 +71,13 @@ mission "First Contact: Hai"
 			`	"So I can travel anywhere I want in your territory?" you ask the Hai.`
 			
 			label travel
+			branch jump
+				not "Discovered Ultima Thule: failed"
 			`	"Yes," it says, "but be careful. The north is the territory of some renegade Hai who are misled in their ways: bandits and pirates, living off what they steal from the rest of us. If you travel among them, you will not be safe."`
 				goto end
+			label jump
+			`	"Yes," it says, "but be careful. The north is the territory of some renegade Hai who are misled in their ways: bandits and pirates, living off what they steal from the rest of us. And if what you say about having a jump drive is true, I would be doubly wary. I have heard rumors that the 'Unfettered Hai' are soliciting captains who land on their worlds to give them jump drives."`
```

Comment:
The fact that the old Hai only tells you this if you arrived by jump drive doesn't make sense because he has no way of knowing that this would be relevant information to you because he doesn't know that you arrived via jump drive.

I think it would make more sense if he always tells you about the Unfettered soliciting jump drives, and if you don't have one yourself yet, you can ask him "What's a jump drive?" or whatever.